internal/operators/ocs/consts.go

```diff
@@ -3,10 +3,14 @@ package ocs
 type ocsDeploymentMode string
 
 const (
-	ssdDrive     string            = "SSD"
-	hddDrive     string            = "HDD"
-	minDiskSize  int64             = 5 //5GB is the min disk size for OCS
-	compactMode  ocsDeploymentMode = "Compact"
-	minimalMode  ocsDeploymentMode = "Minimal"
-	standardMode ocsDeploymentMode = "Standard"
+	CPUCompactMode    int64             = 8  // per host requires 8 cpus for OCS in compact mode
+	CPU               int64             = 6  // per host requires 6 cpus for OCS in minimal and standard mode
```

Comment:
Please update the description to `per host requires at least 6 cpus for OCS in minimal mode`

Reply:
Also update the name to `CPUMinimalMode`